torch/_dynamo/output_graph.py

```diff
@@ -884,7 +884,7 @@ def append_prefix_insts():
             self.random_values_var = self.new_var("random_values")
             rand_fn_name = unique_id("__gen_rand_values")
             rand_fn = disable(_get_gen_rand_values_fn(tx.random_calls))
-            self.install_global_unsafe(rand_fn_name, rand_fn)
+            rand_fn_name = self.install_global_once(rand_fn_name, rand_fn)
```

Comment:
The problem with (2) is that the common pattern is to generate the name, then install the global, and then use the name of the global to do something else (e.g. creating a source with it). As a result, we need some way to return the generated name from the `install_global` function.

Reply:
I split install_global_once into two APIs. One of them does the unique_id automatically. Both of these APIs take in a "prefix" (instead of a "name"), so hopefully it'll be less footgun-y.